src/SDKs/CognitiveServices/dataPlane/Personalizer/Personalizer/Microsoft.Azure.CognitiveServices.Personalizer.csproj

```diff
@@ -6,7 +6,7 @@
   <PropertyGroup>
     <PackageId>Microsoft.Azure.CognitiveServices.Personalizer</PackageId>
     <Description>This client library provides access to the Microsoft Cognitive Services Personalizer APIs.</Description>
-    <Version>0.8.0-preview</Version>
+    <Version>1.0.0-preview</Version>
```

Comment:
Are you sure you want 1.0.0-preview?
I would recommend to stay in 0.x.0-preview...

Reply:
good point, changed to 0.9.0 and updated AssemblyInfo.cs too